cli/cmd/init.go

```diff
@@ -279,11 +281,16 @@ func initCmd(cmd *cobra.Command, args []string) (initCommandError error) {
 		if err != nil {
 			return err
 		}
+
+		apiClientWithoutRetries, err = api.NewClient(token.Value, cqapi.WithHTTPClient(&http.Client{}))
```

Comment:
why not `http.DefaultClient`?
```suggestion
		apiClientWithoutRetries, err = api.NewClient(token.Value, cqapi.WithHTTPClient(http.DefaultClient))
```